rules/windows/process_creation/proc_creation_win_renamed_autohotkey.yml

```diff
@@ -0,0 +1,38 @@
+title: Renamed AutoHotkey.EXE Execution
+id: 0f16d9cf-0616-45c8-8fad-becc11b5a41c
+status: test
+description: Detects execution of a renamed autohotkey.exe binary based on PE metadata fields
+references:
+    - https://www.autohotkey.com/download/
+    - https://thedfirreport.com/2023/02/06/collect-exfiltrate-sleep-repeat/
+author: Nasreddine Bencherchali
+date: 2023/02/07
+tags:
+    - attack.defense_evasion
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        - Product|contains: 'AutoHotkey'
+        - Description|contains: 'AutoHotkey'
+        - OriginalFileName:
+            - 'AutoHotkey.exe'
+            - 'AutoHotkey.rc'
+    filter:
+        Image|endswith:
+            - '\AutoHotkey.exe'
+            - '\AutoHotkey32.exe'
+            - '\AutoHotkey32_UIA.exe'
+            - '\AutoHotkey64.exe'
+            - '\AutoHotkey64_UIA.exe'
+            - '\AutoHotkeyA32.exe'
+            - '\AutoHotkeyA32_UIA.exe'
+            - '\AutoHotkeyU32.exe'
+            - '\AutoHotkeyU32_UIA.exe'
+            - '\AutoHotkeyU64.exe'
+            - '\AutoHotkeyU64_UIA.exe'
```

Comment:
I downloaded all the versions from the website and installed them and these are the available names. I will still add one with contains "just in case"